php-fpm/agents/plugins/php_fpm_pools

```diff
@@ -156,18 +156,18 @@ def parse_fpm_config(f):
     """
     Parse php-fpm config, yielding dicts with status socket info
     """
-    config = configparser.ConfigParser()
+    config = configparser.ConfigParser(strict=False)
     config.read_file(f)
 
     for name in config.sections():
         section = config[name]
         if 'listen' not in section:
             continue
 
-        listen = section['listen']
+        listen = section['listen'].strip("'\"")
```

Comment:
I didn't get this part. Why should we strip these 3 symbols?
Also this sequence repeats 3 times in this file, so please extract it to the variable and reuse.

Reply:
@jplitza Do you remember your reasoning behind this?

Reply:
@elcamlost Basically it's just to support quoted parameters in the config file.